app/actors/curation_concerns/actors/apply_order_actor.rb

```diff
@@ -3,11 +3,28 @@ module Actors
     class ApplyOrderActor < AbstractActor
       def update(attributes)
         ordered_member_ids = attributes.delete(:ordered_member_ids)
+        sync_members(ordered_member_ids)
         apply_order(ordered_member_ids) && next_actor.update(attributes)
       end
 
       private
 
+        def sync_members(ordered_member_ids)
+          return true if ordered_member_ids.nil?
+          existing_members_ids = curation_concern.members.select { |k| k.class.to_s != 'FileSet' }.map(&:id)
```

Comment:
I felt like filesets were unnecessary to iterate through when assigning children to parent works. Eventhough they are members, they arent considered "children", so we didn't want them in the list of existing_member_ids